src/test/java/io/vertx/ext/asyncsql/SQLTestBase.java

```diff
@@ -46,6 +46,14 @@
 
 public abstract class SQLTestBase extends AbstractTestBase {
 
+  static final String POSTGRESQL_HOST = System.getProperty("postgresql.host", "localhost").trim();
```

Comment:
Should we have a `POSTGRESQL_SSL_HOST` as well?

Reply:
I think we can introduce it later if we need it.